core/sql/sqlcomp/CmpDescribe.cpp

```diff
@@ -2355,10 +2355,134 @@ short CmpDescribeHiveTable (
       NAString colString(buf);
       Int32 j = i;
       outputColumnLine(space, colString, j);
-    }
+  }
 
   outputShortLine(space, "  )");
 
+  // show hive table partitions and buckets defination.
```

Comment:
Nit. Typo: defination (should be "definition")

Reply:
Fixed